curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-042.md

```diff
@@ -1,34 +1,33 @@
 ---
-id: 6194079d7cbf586eba9539ff
+id: 620175b3710a0951cfa86edf
 title: Step 42
 challengeType: 0
 dashedName: step-42
 ---
 
 # --description--
 
-Your last `.row` element in each section should stand out, as this is your "total" row. The `:last-child` selector allows you to target the matching element that is the final child in a group of siblings.
-
-Create a `.row:last-child` selector and give it a `background-color` property set to `transparent`. Also set the `margin-bottom` property to `30px` to create some extra space between your sections.
+Ensure your years do not get hidden by setting a `z-index` of `999`. Then, give it a `margin` of `0 -2px`, and a `padding` set to `0.5rem calc(1.25rem + 2px) 0.5rem 0`.
```

Comment:
Why is the `z-index` here?
1) @bbsmooth would have something to say about it being positive
2) I cannot see any visual change.

Reply:
because it was in the prototype 🙃

Reply:
In my [codepen demo](https://codepen.io/bbsmooth/pen/oNoXeav) if I remove `z-index` from the years div then the table content flows over it because I have relative positioning on each table in order to position the caption where I wanted it. If you aren't using relative positioning on the table then ya, you probably won't need a `z-index` on that div.